share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
For what it's worth, I'd like to keep it documented, but in a separate section that clearly states “If you use this feature, you are mostly on your own. Only use if you are comfortable debugging any arising issues yourself.” or something similar. Generally, I'd prefer if more stuff was documented, e.g. commands targeted at maintainers only, instead of trying to hide stuff from users.

(Of course there's no point in having that documentation if nobody is willing to maintain it, but I think it could be feasible to require at least an update to the man page whenever a flag, command, or environment variable is introduced/changed in core code.)